app/perf_overlay.conf

```diff
@@ -1 +1,3 @@
 CONFIG_PERFORMANCE_COUNTERS=y
+CONFIG_SYS_HEAP_RUNTIME_STATS=y
+CONFIG_STACK_SENTINEL=y
```

Comment:
Please enable this config option in a separate patch as this is totally different logical change. 

Also, please imperative mood in the commit message.

``e.g: memory: Add support for heap memory profiling``

Reply:
this is enabled in another patch:#6530, so I will drop here, and when profiling build, just need add -d option is fine